### ISSUE TITLE
RN3 Pro Kenzo new kernel sources.

### ISSUE THREAD

**Issue by ghost** [MiCode/Xiaomi_Kernel_OpenSource#87]
Sinse android 6.0.1 from Miui global stable  (8.1.1.0) has been released - Xiaomi must upload kernel sources code and all drivers to github, because of GPL terms and conditions.

**Comment by ghost** [comment]
No, this is not a copy. 'Cause older ones have not the same description and conditions.

**Comment by ghost** [comment]
Unfortunately we will have to wait for kernel update! 😭😭😭

**Comment by NyaomiDEV** [comment]
+1
